Frontend/android-native/app/src/main/java/in/juspay/mobility/MainActivity.java

```diff
@@ -480,12 +482,15 @@ private void initApp(String viewParam, String deepLinkJSON) {
             @Override
             public void onEvent(JSONObject jsonObject, JuspayResponseHandler juspayResponseHandler) {
                 Log.d(LOG_TAG, "onEvent: " + jsonObject.toString());
+                long initiateTimeStamp = System.currentTimeMillis();
```

Comment:
initiate timestamp should be outside onEvent, it should be when we about to initiate hyperservice